### PR TITLE
Modify test cases due to update of drop index

### LINKED ISSUE #20222: [Bug]: [Nightly]Drop index repeatedly raised AssertionError
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:a2d5947
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev69
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Drop index repeatedly raised AssertionError.

### Expected Behavior

PASSED

### Steps To Reproduce

```markdown
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/179/pipeline/123/
```

### Milvus Log

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by NicoYuan1986**
Resolve: #20222
Signed-off-by: nico <cheng.yuan@zilliz.com>

### CODE REVIEW

File: tests/python_client/testcases/test_index.py [hunk 1]
```diff
@@ -909,8 +909,7 @@ def test_drop_index_repeatedly(self, get_simple_index):
             assert len(collection_w.indexes) == 1
             collection_w.drop_index(index_name=ct.default_index_name)
             assert len(collection_w.indexes) == 0
-            collection_w.drop_index(index_name=ct.default_index_name, check_task=CheckTasks.err_res,
-                                    check_items={ct.err_code: 0, ct.err_msg: "Index doesn\'t exist."})
+            collection_w.drop_index(index_name=ct.default_index_name)
```
Comment (binbinlv) [inline tests/python_client/testcases/test_index.py:912]: Assert again after drop index is more preciseness.

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_index.py:912]: > Assert again after drop index is more preciseness.

Thanks for your advice. Modified.

**Pull request merged**